### PR TITLE
Load Config Values for correct Storeview

### PR DESCRIPTION
**Pull request by dh1984**
Please make sure these boxes are checked before submitting your PR - thank you!

- [x] Pull request is based against develop branch
- [x] README.md reflects changes (if applicable)
- [x] New files contain a license header

### Issue

This PR fixes issue # .

### Proposed changes

...

### CODE REVIEW

File: Model/System/Config.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function isCountryInEu($countryCode)
      */
     public function getEuCountries()
     {
-        $euCountries = $this->scopeConfig->getValue('general/country/eu_countries');
+        $euCountries = $this->scopeConfig->getValue('general/country/eu_countries', 'store');
```
Comment (sprankhub) [inline Model/System/Config.php]: Please use `\Magento\Store\Model\ScopeInterface::SCOPE_STORE` instead.

**Review by sprankhub — CHANGES_REQUESTED** [review]
While you are at it, would you mind changing `ScopeInterface::SCOPE_STORES` to `ScopeInterface::SCOPE_STORE` in `\FireGento\MageSetup\Block\Imprint\Content::getImprintValue` as well?

File: Model/System/Config.php (hunk 1 above)
Comment (dh1984) [reply Model/System/Config.php]: I have changed it and also file \FireGento\MageSetup\Block\Imprint\Content

**Review by sprankhub — APPROVED** [review]

**Comment by sprankhub** [comment]
I am pretty sure the Travis issue is unrelated to this PR, so I will merge it anyway.

**Pull request merged**